doc/help/kdb-record.md

```diff
@@ -0,0 +1,144 @@
+# kdb-record(1) -- Session recording for Elektra
+
+## SYNOPSIS
+
+`kdb record <COMMAND>`
+
+Where `<COMMAND>` is one of the following:
+
+- `start [session_name]`: start or resume a recording session.
+- `stop`: stop the active recording session.
+- `reset [session_name]`: remove all logged data from the recording session.
+- `delete [session_name]`: delete the recording session.
+- `mark [--meta meta_name] <key_name>`: marks a key to be used for validation.
+- `list`: list all available recording sessions.
+- `name`: prints the name of the currently active session.
+- `changes [session_name]`: shows a human-readable list of changes.
+- `export  [--replay|--diff] [session_name]`: creates a file with the changes in the session.
+- `import [--disable-validation] <file_name>`: imports files created by the `export` command.
+
+## DESCRIPTION
+
+The `kdb record` utility is used to manage session recordings.
+Only a single recording session can be active per user at a time.
+A user can have multiple non-active sessions.
+
+If a session is active, every change to the keys, values and meta-data stored in Elektra will be logged into that session.
+
+## ENVIRONMENT VARIABLES
+
+- `ELEKTRA_RECORD_ACTIVE_SESSION`: If this environment variable is set, the specified session will be the active session.
```

Comment:
```suggestion
- `ELEKTRA_RECORD_SESSION`: If this environment variable is set, the specified session will be the active session.
```

Personally, I prefer shorter names for environment variables, but `ELEKTRA_RECORD_ACTIVE_SESSION` would be fine too.